lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -8,6 +8,10 @@ class MarriageAbroadDataQuery
 
     FRENCH_OVERSEAS_TERRITORIES = %w(french-guiana french-polynesia guadeloupe martinique mayotte new-caledonia reunion st-pierre-and-miquelon wallis-and-futuna)
 
+    CEREMONY_COUNTRIES_OFFERING_PACS = %w(
+      france monaco new-caledonia wallis-and-futuna
+    )
+
```

Comment:
This was because I was trying to avoid the long lines that the existing style leads to. However, consistency is more important so I've changed these in https://github.com/alphagov/smart-answers/pull/2470/commits/80e8c02950f22a5da52da69b1697c588ca2aa1fa.